feature/community/src/main/java/com/iteneum/community/presentation/CommunityComposable.kt

```diff
@@ -0,0 +1,60 @@
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.grid.GridCells
+import androidx.compose.foundation.lazy.grid.LazyVerticalGrid
+import androidx.compose.foundation.lazy.grid.items
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import androidx.lifecycle.viewmodel.compose.viewModel
+import com.iteneum.community.R
+import com.iteneum.community.data.CommunityViewModel
+import com.iteneum.community.domain.CardMenuItem
+import com.iteneum.designsystem.components.LpIconTextCard
+import com.iteneum.designsystem.theme.LeasePertTheme
+
+@Composable
+fun CommunityUI(viewModel: CommunityViewModel = viewModel()) {
+    val width = LeasePertTheme.sizes.extraSize124
+    val height = LeasePertTheme.sizes.extraSize128
```

Comment:
Please create just one variable for all the sizes like: `val sizes = LeasePertTheme.sizes`
and when you need to use the `sizes do: sizes. middleSize`

Reply:
booth vals are need width and height are not the same width is more big than height  just confirm that  button will be square and i´ll fix it